sites/www.zooniverse.org.conf

```
@@ -83,6 +83,10 @@ server {
         return 301 /projects/arolsen-archives/every-name-counts$1$2$is_args$query_string;
     }
 
+    location ~* ^/projects//kevinesolberg/mapping-prejudice(/?)(.*?)\/?$ {
```

remove the extra path `/`
```suggestion
    location ~* ^/projects/kevinesolberg/mapping-prejudice(/?)(.*?)\/?$ {
```

---

Oops. Thanks for catching